.babelrc

```diff
@@ -1,4 +1,5 @@
 {
+  "plugins": ["transform-es2015-modules-umd"],
```

Comment:
If I understood correctly, you want me to configure modules: ["umd"] inside env.production? If I do that, babel throws the following exception:

> ReferenceError: [BABEL] src/js/bootstrap-switch.js: Using removed Babel 5 option: /workdir/.babelrc.env.production.modules - Use the corresponding module transform plugin in the `plugins` option. Check out http://babeljs.io/docs/plugins/#modules